### PR TITLE
Not overwrite git_ref in CoprBuildEvent

### LINKED ISSUE #495: assert ref is github.GithubObject.NotSet or isinstance(ref, str), ref
See 
- https://github.com/packit-service-testing-organisation/rpm-source-git/pull/1
- ~~https://sentry.io/organizations/red-hat-0p/issues/1566953230~~ https://sentry.io/organizations/red-hat-0p/issues/1559923043 (edit by @lachmanfrantisek )

```bash
[ERROR/ForkPoolWorker-1] Task task.steve_jobs.process_message[31ee9728-822a-49e8-9aa4-ce537c159ad2] raised unexpected: AssertionError(None)
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/celery/app/trace.py", line 385, in trace_task
   R = retval = fun(*args, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/celery/app/trace.py", line 650, in __protected_call__
   return self.run(*args, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/sentry_sdk/integrations/celery.py", line 169, in _inner
   reraise(*exc_info)
  File "/usr/local/lib/python3.7/site-packages/sentry_sdk/_compat.py", line 57, in reraise
   raise value
  File "/usr/local/lib/python3.7/site-packages/sentry_sdk/integrations/celery.py", line 164, in _inner
   return f(*args, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/packit_service/worker/tasks.py", line 42, in process_message
   return SteveJobs().process_message(event=event, topic=topic)
  File "/usr/local/lib/python3.7/site-packages/packit_service/worker/jobs.py", line 323, in process_message
   jobs_results = self.process_jobs(event_object)
  File "/usr/local/lib/python3.7/site-packages/packit_service/worker/jobs.py", line 186, in process_jobs
   handlers_results[job.type.value] = handler.run_n_clean()
  File "/usr/local/lib/python3.7/site-packages/packit_service/worker/handlers/abstract.py", line 109, in run_n_clean
   return self.run()
  File "/usr/local/lib/python3.7/site-packages/packit_service/worker/handlers/fedmsg_handlers.py", line 293, in run
   chroot=self.event.chroot,
  File "/usr/local/lib/python3.7/site-packages/packit_service/worker/handlers/github_handlers.py", line 603, in __init__
   self.project, self.base_ref, pr_id
  File "/usr/local/lib/python3.7/site-packages/packit_service/config.py", line 146, in get_package_config_from_repo
   project, reference
  File "/usr/local/lib/python3.7/site-packages/packit/config/package_config.py", line 271, in get_package_config_from_repo
   path=config_file_name, ref=ref
  File "/usr/lib/python3.7/site-packages/ogr/services/github/project.py", line 386, in get_file_content
   path=path, ref=ref
  File "/usr/lib/python3.7/site-packages/github/Repository.py", line 1504, in get_contents
   assert ref is github.GithubObject.NotSet or isinstance(ref, str), ref
  AssertionError: None
```

### PR DESCRIPTION
**Pull request by lbarcziova**
May fix #495

### CODE REVIEW

File: packit_service/service/events.py [hunk 1]
```diff
@@ -514,20 +514,18 @@ def __init__(
...
             self.pr_id = build_pg.pr.pr_id
             self.commit_sha = build_pg.commit_sha
-            self.git_ref = (
-                self.commit_sha
-            )  # ref should be name of the branch, not a hash
             self.base_repo_name = build_pg.pr.project.repo_name
             self.base_repo_namespace = build_pg.pr.project.namespace
             # FIXME: hardcoded, move this to PG
             https_url = f"https://github.com/{self.base_repo_namespace}/{self.base_repo_name}.git"
+            git_ref = self.commit_sha  # ref should be name of the branch, not a hash
```
Comment (lachmanfrantisek) [inline packit_service/service/events.py]: Moving down does not help, but the rewriting will help definitely.

An alternative solution can be to move the `super` call up and safely write these values after that. What do you think?

Comment (lbarcziova) [reply packit_service/service/events.py]: yes, that was my first idea, but we need to get `https_url` from the db info as it is passed to the `super` constructor

Comment (lachmanfrantisek) [reply packit_service/service/events.py]: Good point. Then maybe only the `https_url` before the `super` calls.

(Feel free to merge it as is. I just don't like a lot of code before the super call. It usually leads to this kind of weird errors...)

Comment (lbarcziova) [reply packit_service/service/events.py]: I will leave it as it is for now, as here `https_url = f"https://github.com/{self.base_repo_namespace}/{self.base_repo_name}.git"` we need other attributes anyway.

**Pull request merged**